provisioner/shell/provisioner.go

```diff
@@ -430,21 +436,24 @@ func (p *Provisioner) escapeEnvVars() ([]string, map[string]string) {
 func (p *Provisioner) createEnvVarFileContent() string {
 	keys, envVars := p.escapeEnvVars()
 
-	flattened := ""
-	// Re-assemble vars surrounding value with single quotes and flatten
+	var flattened string
+	line := fmt.Sprintf("export %s\n", p.config.EnvVarFormat)
```

Comment:
Should the `env_var_format` be limited to just the key,value format? Or would a user expect it pertain to the entire string template (e.g `"export %s='%s'\n"`) when `use_env_var_file` is `true`?

Reply:
I'd say the entire string template, which we can default to containing "export" when the use_env_var_file is true. That gives the most flexibility while keeping the default consistent.

Reply:
Thanks @SwampDragons that helps a lot.